src/tinohelm/backtest/runner.py

```
@@ -1007,70 +980,160 @@ def _build_funding_rate_updates(funding_events: list[dict]) -> list:
         """
-        from tinohelm.data.catalog import CatalogSession
+        from nautilus_trader.model.data import FundingRateUpdate, MarkPriceUpdate
+        from nautilus_trader.model.identifiers import InstrumentId
         from tinohelm.data.funding_cache_helpers import compute_fetch_start
-        from tinohelm.data.funding_cache import _load_cache
         from tinohelm.data.instruments import fetch_funding_info, strip_to_binance_api_symbol
 
-        session = CatalogSession(self.catalog_path, storage=self._storage)
+        catalog = self._catalog_for_path(self.catalog_path)
+        funding_info = fetch_funding_info()
+        nt_symbols_by_symbol: dict[str, str] = dict(zip(self.symbols, nt_symbols))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)